securedrop/db.py

```diff
@@ -190,6 +193,18 @@ class BadTokenException(Exception):
     """Raised when a user logins in with an incorrect TOTP token"""
 
 
+class InvalidPasswordLength(Exception):
+    """Raised when attempting to create a Journalist or log in with an invalid
+    password length"""
+
+    def __init__(self, password):
+        self.password = password
```

Comment:
Just seemed more flexible for future functionality, but could also just store the length to be safe.